docs/key-issues/multiple_architectures.md

```diff
@@ -0,0 +1,344 @@
+      [py2app](https://py2app.readthedocs.io) or
+      [briefcase](https://beeware.org/project/projects/tools/briefcase/)),
+      the person building the project must be accommodate all possible CPU
+      architectures where the code *could* be executed.
+    - If binary wheels are only available in "thin" format, any issues with
+      merging those wheels into fat equivalents for distribution purposes are
+      deferred to the end user. This can be problematic as it may require
+      expert knowledge about the package being merged (such as optional
+      modules or header files that may not be present in both thin artefacts).
+      Universal2 artefacts captures this knowledge by requiring the project
+      maintainers to resolve any merging issues.
+
+    Arguments against:
+
+    - `universal2` wheels are never necessary for end users installing into
+      a locally installed Python environment exclusively for their own use,
+      which is the default experience most users have with Python.
+    - The tradeoff between download size and disk space usage vs. the upside
+      for say a .dmg installer is bad - for a typical PyData stack it takes
+      hundreds of MBs per Python environment more than thin wheels, and users
+      are likely to have quite a few environments on their system at once.
+      Meaning that defaulting to `universal2` would use several GBs of disk
+      space more.
+
+        - Disk space on older MacBook Air models is 128 GB, and up to half of that
+          can be taken up by the OS and system data itself. So a few GBs can be
+          significant.
+        - Internet plans in many countries are not unlimited; almost doubling the
+          download size of wheels is a serious cost, and not desirable for any
+          user - but especially unfriendly to users in countries where network
+          infrastructure is less developed.
+
+    - In addition, it takes extra space on PyPI (examples: `universal2` wheels
+      cost an extra 81.5 MB for NumPy 1.21.4 and 175.5 MB for SciPy 1.9.1), and
+      projects with large wheels often run into total size limits on PyPI.
+    - It imposes an extra maintenance burden for each project, because separate
+      CI jobs are needed to build and test `universal2` wheels. Typically
+      projects make tradeoffs there, because they cannot support every
+      platform. And `universal2` doesn't meet the bar for usage frequency /
+      user demand here - it's well below the demand for `musllinux`,
+      `ppc64le`, PyPy, and other such platforms with still patchy support
+      (see [Expectations that projects provide ever more wheels](../../meta-topics/user_expectations_wheels.md)
+      for more on that).
+    - When a project provides thin wheels (which is a must-do for projects with
+      native code, because those are the better experience due to smaller
+      size), you cannot even install a `universal2` wheel with pip from PyPI at
```

Comment:
Again, subjective, use-case specific language. In my use case, universal2 wheels unequivocally provide a better experience.

Reply:
"because those are the better experience with pip install into an active environment" perhaps?

Because it *is* unequivocally better for all use cases where you aren't going to transfer the results to another computer. And virtual environments are not relocatable. And I'd imagine that conda doesn't provide a universal2 interpreter so conda environments *might* be relocatable, but presumably not across machines.

Reply:
>  And I'd imagine that conda doesn't provide a universal2 interpreter so conda environments might be relocatable, but presumably not across machines.

Conda environments should be more or less relocatable (not that I'd recommend it), but only within the same CPU architecture. Everything in conda is per-arch, there are no fat binaries or something like universal2.

Reply:
> "because those are the better experience with pip install into an active environment" perhaps?

That sounds fine to me. Maybe adding "as an end user"? I imagine `briefcase` and `py2app` also use venv's under the hood.

Reply:
I was thinking that those would be inactive environments, but it's a subtle distinction and "as an end user" is probably a better one.

Reply:
FWIW, Briefcase doesn't use literal venvs, but it does do some venv-like tricks to ensure interpreter isolation. I can't speak to the current state of py2app.

Reply:
I toned this down from "must do" to "should be done" and added the clarification that it's only better when installing for use on that machine. Please resolve this comment if that looks good.